jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -360,6 +361,12 @@ const DeploymentFlow = React.createClass({
       'Button click',
       'Deploy model'
     );
+    if (this.props.stats) {
+      const flags = window.juju_config.flags;
+      const statName = 'deploy' + (flags.ddeploy ? '.direct' : '.target');
```

Comment:
If you need to know if this is a direct deploy DF the prop `ddData` will have been populated.